roles/cm_agents/tasks/main.yml

```diff
@@ -2,6 +2,7 @@
 
 - name: Apply temporary workaround for Cloudera Manager issue OPSAPS-36322
   include: 36322.yml
+  when: cluster_install_groups_users | default(false)
```

Comment:
Btw @dbeech is this 'temporary' workaround still needed in the playbook (maybe for older CDH 5.x versions ?).
During my initial tests with CDH 6.2, I had this part ignored, and the deployment worked (at least upto the CM install + configuration)

Reply:
No, I don't think this is required any more. The bug listed is 3 years old and wasn't even commonly encountered -- I think the original root cause was that selinux was enabled and this messed up the user creation through CM.

Reply:
OK, then I propose to get that whole logic out in a future PR
( but keep my addition in for now, assuming you are OK with the rest)